pluginkit/test_suite.go

```diff
@@ -95,76 +96,67 @@ func (t *TestSuite) AddTestSetResult(name string, result TestSetResult) {
 	t.TestSetResults[name] = result
 }
 
-// WriteTestSetResultsJSON unmarhals the TestSuite into a JSON file in the user's WriteDirectory
-func (t *TestSuite) WriteTestSetResultsJSON() error {
-	// Log an error if PluginName was not provided
-	if t.TestSuiteName == "" {
-		return errors.New("TestSuite name was not provided before attempting to write results")
+func (t *TestSuite) WriteTestSetResults(serviceName string, output string) error {
+	if t.TestSuiteName == "" || serviceName == "" {
+		return fmt.Errorf("testSuite name and service name must be provided before attempting to write results: testSuite='%s' service='%s'", t.TestSuiteName, serviceName)
 	}
-	filepath := path.Join(t.config.WriteDirectory, t.TestSuiteName, "results.json")
```

Comment:
@eddie-knight I think this is ready for review, but was curious why we previously had JSON results in result.json but YAML results were [testSuiteName].yaml? iterative coding or on purpose?